src/main/java/org/ow2/proactive/catalog/report/TableDataBuilder.java

```diff
@@ -129,9 +163,10 @@ private String getProjectName(CatalogObjectMetadata catalogObject) {
     private String getDescription(CatalogObjectMetadata catalogObject) {
         return catalogObject.getMetadataList()
                             .stream()
-                            .filter(metadata -> metadata.getKey().equals("description"))
+                            .filter(metadata -> metadata.getLabel().equals("General") &&
+                                                metadata.getKey().equals("description"))
                             .map(metadata -> metadata.getValue())
-                            .map(description -> description.replaceAll("\n", "").replace("\r", "").replace("\t", ""))
+                            .map(description -> description.replaceAll("\n", " ").replace("\r", "").replace("\t", ""))
```

Comment:
It would be good to change the method name here for the same reason as the upper comment.